doc/api/test.md

```diff
@@ -678,11 +678,9 @@ added:
   properties are supported:
   * `concurrency` {number|boolean} If a number is provided,
     then that many files would run in parallel.
-    If truthy, it would run (number of cpu cores - 1)
-    files in parallel.
-    If falsy, it would only run one file at a time.
-    If unspecified, subtests inherit this value from their parent.
-    **Default:** `true`.
+    If `true`, it would run `os.availableParallelism() - 1` tests in parallel.
+    If `false`, it would only run one test at a time.
```

Comment:
```suggestion
    If `true`, it would run `os.availableParallelism() - 1` test files in parallel.
    If `false`, it would only run one test file at a time.
```